addons/mail/views/mail_activity_views.xml

```diff
@@ -194,6 +194,25 @@
         </field>
     </record>
 
+    <record id="mail_activity_view_form_without_record_access" model="ir.ui.view">
```

Comment:
Was the existing form view not appropriate? We are not even overriding it? Maybe there is too much stuff we don't want and it's easier to re-define a whole view but it seems a bit extreme ^^

Reply:
Actually, after testing a bit, we can’t use any action that “mark as done” the activity because it make the client crash because it display an activity that doesn’t exit anymore then. So I can override it and remove every action. We can’t also open the document as the user as no access to it so I would also have to remove that. And I thought that addons will probably add think that the user has no access to. That’s why I have defined a brand new simple view. What do you think ? Do you prefer I override it and remove all that things ?

Reply:
It's fine, keep it like this then.